code/core/src/common/presets.ts

```diff
@@ -12,12 +12,12 @@ import type {
   StorybookConfigRaw,
 } from 'storybook/internal/types';
 
-import { parseNodeModulePath } from 'mlly';
```

Comment:
`mlly` is not deprecated. Only resolution util, you can keep using `mlly` for `parseNodeModulePath` :)